src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -93,55 +93,79 @@ export const CarSharingStationSheet = ({stationId, distance, close}: Props) => {
+                          color="secondary"
+                        >
+                          {t(
+                            CarSharingTexts.stations.carsAvailable(
+                              totalAvailableCars(station.vehicleTypesAvailable),
+                              station.capacity,
+                            ),
+                          )}
+                        </ThemeText>
+                      </View>
+                      <ThemeText type="body__secondary" color="secondary">
+                        {t(CarSharingTexts.stations.carsAvailableLabel)}
+                      </ThemeText>
+                    </View>
+                    <View style={style.carDetailsContainer}>
+                      {station.vehicleTypesAvailable
+                        ?.slice(station.capacity > 2 ? -1 : -2)
+                        .map((vehicle, i) => (
+                          <View
+                            key={i}
+                            style={[
+                              style.carImage,
+                              i === station.capacity - 1
+                                ? style.carImageLast
+                                : {},
+                            ]}
+                          >
                             <CarImage uri={vehicle.vehicleType.vehicleImage} />
                           </View>
-                          <View style={style.carDetails}>
-                            <CarName vehicleType={vehicle.vehicleType} />
-                            <ThemeText type="body__secondary">
-                              {t(
-                                CarSharingTexts.propultionType(
-                                  vehicle.vehicleType.propulsionType,
-                                ),
-                              )}
-                            </ThemeText>
-                            <View style={style.availabilityChip}>
-                              <InfoChip
-                                text={t(
-                                  CarSharingTexts.stations.carsAvailable(
-                                    vehicle.count,
-                                  ),
-                                )}
-                                interactiveColor="interactive_0"
-                              />
-                            </View>
-                          </View>
+                        ))}
+                      {station.capacity > 2 && (
+                        <View
+                          style={{
+                            height: 40,
+                            width: 40,
+                            borderRadius: 8,
+                            backgroundColor: '#d8d8d8',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                          }}
+                        >
+                          <ThemeText style={{justifyContent: 'center'}}>
+                            +{station.capacity - 1}
```

Comment:
Mismatch in logic between Figma and app. In Figma the plus number is only the remaining available, but the plus number in the app is the remaining capacity. Not sure which is correct, but it should match at least.
<div>
<img width="432" src="https://github.com/AtB-AS/mittatb-app/assets/675421/cb1d0b44-23b1-4d5c-8659-0bf2bab95e24"/>
<img width="432" src="https://github.com/AtB-AS/mittatb-app/assets/675421/78ed5de6-066d-4a06-972c-e6d64de18519"/>
</div>

Reply:
Figma updated with proper documentation, the plus should show the total cars, even if there is no available cars.

https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?type=design&node-id=20471-12079&mode=design&t=QlA6jv5flbjP5Iyt-4